addons/escoria-core/game/core-scripts/log/esc_logger.gd

```diff
@@ -85,13 +116,21 @@ func warning(string: String, args = []):
 # 
 # * string: Text to log
 # * args: Additional information
-func error(string: String, args = []):
+func error(string: String, args = [], do_savegame: bool = true):
 	if _get_log_level() >= LOG_ERROR:
 		var argsstr = str(args) if !args.empty() else ""
-		printerr("(E)\t" + string + " \t" + argsstr)
+		_log("(E)\t" + string + " \t" + argsstr, true)
 		if ProjectSettings.get_setting("escoria/debug/terminate_on_errors"):
-			print_stack()
+			_log(
+				ProjectSettings.get_setting("escoria/debug/crash_message") 
```

Comment:
The crash message should include the path to the log file and the save game. Like "We're sorry, but the game crashed. Please send us the following files: (...)"

I saw that you've added the log path in the message, but at this point the specific files could be provided.

Reply:
Fixed